fvtest/tril/test/IlGenTest.cpp

```diff
@@ -50,7 +50,10 @@ TEST_F(IlGenTest, Return3) {
     auto Int32 = types.PrimitiveType(TR::Int32);
     TR::IlType* argTypes[] = { Int32 };
 
-    Tril::TRLangBuilder injector(trees, &types);
+    Tril::CallConverter callConverter;
+    Tril::GenericNodeConverter genericNodeConverter(&callConverter);
```

Comment:
Shouldn't `GenericNodeConverter` be constructed first? Otherwise, it will get the first attempt to convert an `ASTNode`, which will always succeed and never reach the `CallConverter`.